### PR TITLE
Add grant manager email address

### PR DESCRIPTION
**Pull request by r-czajkowski**
`Grant Manager` could be too confusing so we put an email address to
contact the grant manager.

<details>
  <summary>Screenshots</summary>
  
  obraz
obraz
</details>

### CODE REVIEW

File: solidity-v1/dashboard/src/components/modal/staking/ConfirmDelegation.jsx [hunk 1]
```diff
@@ -23,9 +38,15 @@ export const ConfirmDelegation = withBaseModal(
...
-            <p className="text-black mt-1 text-bold">
-              To use this stake in Threshold, you will first have to contact
-              your grant manager. Please do it immediately after you stake.
+            <p style={styles.warning}>
+              Please do not forget to contact your Grant Manager{" "}
+              <a
+                style={styles.grantManager}
+                href={`mailto:${GRANT_MANAGER_EMAIL}`}
+              >
+                ({GRANT_MANAGER_EMAIL})
```
Comment (michalsmiarowski) [inline solidity-v1/dashboard/src/components/modal/staking/ConfirmDelegation.jsx:47]: I think we should move the brackets outside of the `<a>` tag.

File: solidity-v1/dashboard/src/components/modal/threshold/ContactYourGrantManagerWarning.jsx [hunk 2]
```diff
@@ -2,22 +2,42 @@ import React from "react"
...
       <>
-        <ModalHeader>{header}</ModalHeader>
+        <ModalHeader style={styles.header}>{header}</ModalHeader>
         <ModalBody>
           <h3>{bodyTitle}</h3>
+          <p className="text-grerey-70 mt-2">
+            To use this stake in Threshold, you will have to contact your grant
+            manager&nbsp;
+            <a href={`mailto:${GRANT_MANAGER_EMAIL}`}>
+              ({GRANT_MANAGER_EMAIL})
```
Comment (michalsmiarowski) [inline solidity-v1/dashboard/src/components/modal/threshold/ContactYourGrantManagerWarning.jsx:32]: Same as above.

File: solidity-v1/dashboard/src/components/modal/threshold/ContactYourGrantManagerWarning.jsx [hunk 3]
```diff
@@ -2,22 +2,42 @@ import React from "react"
...
+            </a>
+            .
+          </p>
+          <p className="mt-2">Please do it immediately!</p>
         </ModalBody>
         <ModalFooter>
-          <Button className={"btn btn-secondary btn-lg"} onClick={onClose}>
-            close
+          <Button className="btn btn-unstyled text-link" onClick={onClose}>
+            Close
```
Comment (michalsmiarowski) [inline solidity-v1/dashboard/src/components/modal/threshold/ContactYourGrantManagerWarning.jsx:40]: Should be

```suggestion
            Cancel
```

as it is in Figma designs

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]
Left some comments, but overall very good

File: solidity-v1/dashboard/src/components/modal/threshold/ContactYourGrantManagerWarning.jsx (hunk 3 above)
Comment (r-czajkowski) [reply solidity-v1/dashboard/src/components/modal/threshold/ContactYourGrantManagerWarning.jsx]: [244a15e](https://github.com/keep-network/keep-core/pull/2853/commits/244a15ea0b9c2192fc473b26ba4b2e3520724f3a)

File: solidity-v1/dashboard/src/components/modal/threshold/ContactYourGrantManagerWarning.jsx (hunk 2 above)
Comment (r-czajkowski) [reply solidity-v1/dashboard/src/components/modal/threshold/ContactYourGrantManagerWarning.jsx]: [244a15e](https://github.com/keep-network/keep-core/pull/2853/commits/244a15ea0b9c2192fc473b26ba4b2e3520724f3a)

File: solidity-v1/dashboard/src/components/modal/staking/ConfirmDelegation.jsx (hunk 1 above)
Comment (r-czajkowski) [reply solidity-v1/dashboard/src/components/modal/staking/ConfirmDelegation.jsx]: [244a15e](https://github.com/keep-network/keep-core/pull/2853/commits/244a15ea0b9c2192fc473b26ba4b2e3520724f3a)

**Review by michalsmiarowski — APPROVED** [review]
LGTM 🚀

**Pull request merged**